.github/workflows/tests.yml

```diff
@@ -0,0 +1,20 @@
+name: Run Rive tests
+
+on:
+  push:
+  pull_request:
+    branches:
+    - master
```

Comment:
I think just `push` is enough. It should not allow the PR to be merged if the checks fail there.